static/js/components/Dashboard.js

```diff
@@ -1,39 +1,42 @@
 /* global SETTINGS: false */
 import React from 'react';
+import Grid, { Cell } from 'react-mdl/lib/Grid';
 import CourseList from './CourseList';
 import UserImage from './UserImage';
 
 class Dashboard extends React.Component {
   static propTypes = {
     profile:    React.PropTypes.object.isRequired,
     dashboard:  React.PropTypes.object.isRequired,
+    expander: React.PropTypes.object.isRequired,
+    dispatch: React.PropTypes.func.isRequired,
   };
 
   render() {
-    const { profile, dashboard } = this.props;
+    const { profile, dashboard, expander, dispatch } = this.props;
     let imageUrl = (SETTINGS.edx_base_url + '/static/images/profiles/default_120.png').
     //replacing multiple "/" with a single forward slash, excluding the ones following the colon
     replace(/([^:]\/)\/+/g, "$1");
     if (profile.profile_url_large) {
       imageUrl = profile.profile_url_large;
     }
     return <div className="card">
-      <div className="card-user">
-        <div className="card-image-box">
+      <Grid className="card-user">
+        <Cell col={2} className="card-image-box">
             <UserImage imageUrl={imageUrl}/>
-        </div>
-        <div className="card-name">
+        </Cell>
+        <Cell col={5} className="card-name">
           { profile.preferred_name || SETTINGS.name }
```

Comment:
I think this is just for the moment after the page loads but before we get data from the profile API